ecf/rtofs/stats/jevs_rtofs_stats.ecf

```diff
@@ -36,13 +38,14 @@ export STEP=stats
 export COMPONENT=rtofs
 
 # set up VDATE and COMIN and COMOUT
-export VDATE=$(date --date="2 days ago" +%Y%m%d)
+export VDATE=$(date --date="3 days ago" +%Y%m%d)
 
-export COMIN=${COMIN:-$(compath.py $envir/com/$NET/${evs_ver})}
+#export COMIN=${COMIN:-$(compath.py $envir/com/$NET/${evs_ver})}
```

Comment:
This line (43) should be deleted. That construct is not useful in the ecf file since there is no processing upstream. This line should be in the job file, and it is already.

Reply:
Done.